backend/pom.xml

```diff
@@ -300,5 +300,27 @@
 			</plugin>
 		</plugins>
 	</build>
+	<profiles>
+		<profile>
+			<id>dependency-check</id>
+			<build>
+				<plugins>
+					<plugin>
+						<!-- Perform CVE dependency scans during the verify phase -->
+						<groupId>org.owasp</groupId>
+						<artifactId>dependency-check-maven</artifactId>
+						<version>${dependency-check-maven.version}</version>
```

Comment:
A new field within `<properties>` for `dependency-check-maven.version` should be introduced (the latest version is 10.0.4).